packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -34,12 +49,19 @@
   $input-border-alpha: -0;
   $tile-border-color: color("gray-20");
   $tile-border-color--disabled: color("gray-10");
+  $tile-box-shadow--disabled: 0 0 0 units($theme-input-select-border-width)
```

Comment:
## New variable

Created this new variable so we can update the disabled tile box shadow in one location. Affects checked + indeterminate checkboxes, ignores indeterminate radio